README.md

```diff
@@ -109,9 +109,11 @@ client.Write("path.to.num", 42);
 Dead-simple monitoring:
 
 ````cs
-client.Monitor<string>("path.to.string", (newValue, unsubscribe) =>
+client.Monitor<string>("path.to.string", (readEvent, unsubscribe) =>
 {
-  DoSomethingWithYourValue(newValue);
+  DoSomethingWithQuality(readEvent.Quality);
```

Comment:
please remove quality and timestamp from example. This is supposed to be the simplest use-case where the developer just wants to use a value.